fibonacci.js

```diff
@@ -0,0 +1,57 @@
+let getNumber = document.getElementById("numberHere");
+let fibURL = "http://localhost:5050/fibonacci/";
+let answer = document.getElementById("answer");
+let mySpinner = document.getElementById("spinner")
+
+function getFib() {
+  mySpinner.classList.remove("visually-hidden");
+  answer.classList.add("visually-hidden");
+
+  console.log(`${fibURL}${getNumber.value}`);
+
+  fetch(`${fibURL}${getNumber.value}`)
+    .then(function (response) {
+      if (!response.ok) {
+        response.text().then((error) => {
+          answer.classList.remove("visually-hidden");
+          answer.classList.add("text-danger");
+          getNumber.classList.add("border-danger");
+          answer.innerText = error;
+          mySpinner.classList.toggle("visually-hidden");
+          console.log(error);
+        });
+      }
+      return response.json();
```

Comment:
in case you send "42" to the server and recive an error, why return response.json()? maybe use else {}